registry.json

```diff
@@ -1727,5 +1727,20 @@
       "telegram":"manizzler",
       "operatorName":"manizzler",
       "github":"manizzle"
+   },
+   {
+      "ticker":"GPOOL",
+      "poolId":"eb6e1949ef2e1f2cdb9b041d2a4b86d4d837798ab3ddfe53cc4cafe4",
```

Comment:
poolid is mapping to ADATA on PoolTool.

Reply:
yes sorry i thought i updated the pool request correctly but didn't. 
We renamed the pool some days ago. Will do now